packages/server/src/modules/product/ProductModel.ts

```diff
@@ -0,0 +1,50 @@
+import mongoose, { Document, Model } from 'mongoose';
+
+const { ObjectId } = mongoose.Schema.Types;
+const Schema = mongoose.Schema;
+
+const ProductSchema = new Schema(
+  {
+    name: {
+      type: String,
+      required: 'name is required',
+    },
+    description: {
+      type: String,
+      required: false,
+    },
+    pictureUrl: {
+      type: String,
+      required: false,
+    },
+    points: {
+      type: Number,
+      required: false,
+    },
+    storeId: {
+      type: ObjectId,
+      required: false,
+    },
```

Comment:
you need to and a ref field to relate these collections.

```suggestion
    storeId: {
      type: ObjectId,
      ref: 'Store',
      required: false,
    },